opm/io/eclipse/ESmry.hpp

```diff
@@ -27,8 +27,14 @@ namespace Opm { namespace EclIO {
 class ESmry
 {
 public:
-    explicit ESmry(const std::string& filename, bool loadBaseRunData=false);   // filename (smspec file) or file root name
 
+    // there is no information in the SMSPEC file with regards to unified or multiple results files.
+    // if unified = true, but UNRST file not found, the constructor will check for non-unified (multiple) 
```

Comment:
`UNRST` &rightarrow; `UNSMRY`

Reply:
this comment is actually outdated, has been deleted. keeping comment in ESmry.cpp file.